src/workerd/api/global-scope.c++

```diff
@@ -455,6 +455,17 @@ kj::Promise<WorkerInterface::AlarmResult> ServiceWorkerGlobalScope::runAlarm(
       KJ_IF_SOME(status, context.getLimitEnforcer().getLimitsExceeded()) {
         outcome = status;
       }
+
+      // We want to alert if we aren't going to count this alarm retry against limits
+      if (auto desc = e.getDescription();
+          !jsg::isTunneledException(desc) && !jsg::isDoNotLogException(desc)
+          && context.isOutputGateBroken()) {
+        if (isInterestingException(e)) {
+          LOG_EXCEPTION("alarmOutputLock"_kj, e);
```

Comment:
Do you have a sense of how often this will happen?  Will we be spamming Sentry?

Reply:
Hopefully it won't happen much. I can edit and make it always NO_SENTRY. We mostly want this for debugging purposes